management-web/src/main/webapp/WEB-INF/pages/product.jsp

```diff
@@ -0,0 +1,22 @@
+<%@include file="../commons/header.jsp" %>
+<%@include file="../commons/navigation.jsp" %>
+<form:form method="post" commandName="product">
+    <fieldset class="form-group">
+        <form:label path="name">Name:</form:label>
+        <form:input path="name" type="text" class="form-control"
+                    required="required"/>
```

Comment:
Research about the support for this in the different browsers. The required property is fine, however, don't trust just in this, in future versions you could add JS validations as well as backend validations in order to mitigate any possible conflict